elide-async/src/main/java/com/yahoo/elide/async/service/DefaultAsyncQueryDAO.java

```diff
@@ -163,22 +165,21 @@ private AsyncQuery updateAsyncQuery(AsyncQuery asyncQuery, UpdateQuery updateFun
     }
 
     @Override
-    public AsyncQueryResult createAsyncQueryResult(Integer status, String responseBody,
+    public AsyncQuery createAsyncQueryResult(Integer status, String responseBody,
```

Comment:
This method should be renamed as per the comment on Async should get the AsyncQuery and AsyncQueryResult object. We set the asyncQueryResult in AsyncQueryObject and then save it.